modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -425,6 +425,13 @@ sda.enkf.multisite <- function(settings,
           }) %>%
           stats::setNames(site.ids)
         
+        #if it's a rabbitmq job sumbmission, we will first copy and paste the whole run folder within the SDA to the remote host.
+        if (!is.null(settings$host$rabbitmq)) {
```

Comment:
1. reminder that eventually we'll have to have each pod "pull" the data
2. the details below are not general, they are very specific to SIPNET and NERC. Is "sipnet.label" something that is potentially actually "model.label"? Can we make some of this part of the configuration (e.g. put some of it in settings$host$rabbitmq) so that if you're on a different cloud system it could be configured differently (e.g. I think "oc" is RH OpenShift specific)

Reply:
I simplified the arguments in the settings into two parts: 1) copy to remote command and 2) copy to local command. We need to replace the `@OUTDIR@` and `@RUNDIR@` with the current paths and run the function.